### ISSUE TITLE
Offer option to create ".nomedia" if the user adds a Syncthing folder

### ISSUE THREAD

**Issue by Catfriend1** [Catfriend1/syncthing-android#769]
Purpose:

Give the user the option to not let Android's media scanner run through the synced folder contents. Some users sync non camera picture files where every JPG/PNG/BMP/etc. asset of a synced app/game/etc. will show up in the gallery without the ".nomedia" file in place.

Ref.: https://forum.syncthing.net/t/how-to-hide-syncthing-folder-in-android/16602/3?u=catfriend1

**Comment by domportera** [comment]
yes please, every time i add a .nomedia file it gets moved to a .thumbnails folder or deleted

**Comment by Catfriend1** [comment]
> yes please, every time i add a .nomedia file it gets moved to a .thumbnails folder or deleted

I don't think it is Syncthing doing this.

**Comment by domportera** [comment]
i cant say for sure but for whatever reason when i put a .nomedia in the root synced folder it gets deleted (in retrospect definitely not moved, i was tired). i can test it further though and if i can reproduce it with other syncthing folders ill update this thread or start a bug report

**Comment by Catfriend1** [comment]
The best way to manually apply is to rename the affected folder in Material Files and rename it back. Gallery shows the contents no longer if .nomedia was added.

**Comment by Catfriend1** [comment]
+ .nomedia in the .stversions folder

**Comment by Catfriend1** [comment]
If you'd like to implement this, here is a good guideline on how to write the java code to create ".nomedia" using SAF/DocumentsProvider or absolute paths using java.File API.

ref: https://github.com/Catfriend1/syncthing-android/pull/1397
